File: examples/indexer/javascript/AssetsBalancesMinBalance.js
// AssetsBalancesMinBalance.js
// requires algosdk@1.6.1 or higher 
// verify installed version
// npm list algosdk

const algosdk = require('algosdk');
const indexer_token = "";
const indexer_server = "http://localhost";
const indexer_port = 8980;

// Instantiate the indexer client wrapper
let indexerClient = new algosdk.Indexer(indexer_token, indexer_server, indexer_port);

(async () => {
    let assetIndex = 2044572;
    let currencyGreater = 200;
    let assetInfo = await indexerClient.lookupAssetBalances(assetIndex)
        .currencyGreaterThan(currencyGreater).do();
    console.log("Information for Asset: " + JSON.stringify(assetInfo, undefined, 2));
})().catch(e => {
    console.log(e);
    console.trace();
});

File: examples/indexer/python/assets_balances_min_balance.py
# assets_balances_min_balance.py
import json
# requires Python SDK version 1.3 or higher
from algosdk.v2client import indexer

# instantiate indexer client
myindexer = indexer.IndexerClient(indexer_token="", indexer_address="http://localhost:8980")

# gets assets with a min balance of 200 for AssetID
response = myindexer.asset_balances(
    asset_id=2044572, min_balance=200)

print("Asset Balances :" + json.dumps(response, indent=2, sort_keys=True))
